### PR TITLE
style(crons): Add docs to monitorcheckin CheckInStatus

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/models/monitorcheckin.py [hunk 1]
```diff
@@ -16,12 +16,22 @@
...
+    """Checkin failed or otherwise had some issues"""
+
     IN_PROGRESS = 3
+    """Checkin is expected to complete"""
+
     MISSED = 4
+    """Monitor did not check in on time"""
 
     FINISHED_VALUES = (OK, ERROR)
+    """Sentient values used to indicate a monitor is finished running"""
```
Comment (wedamija) [inline src/sentry/models/monitorcheckin.py:34]: Sentinel?

**Review by wedamija — APPROVED** [review]

File: src/sentry/models/monitorcheckin.py [hunk 2]
```diff
@@ -16,12 +16,22 @@
 
 class CheckInStatus:
     UNKNOWN = 0
+    """No status was passed"""
```
Comment (wedamija) [inline src/sentry/models/monitorcheckin.py:19]: Feels like these should be above or inline, and just use `#`

Comment (evanpurkhiser) [reply src/sentry/models/monitorcheckin.py:19]: But this puts them into the doc comments
image

File: src/sentry/models/monitorcheckin.py (hunk 1 above)
Comment (evanpurkhiser) [reply src/sentry/models/monitorcheckin.py:34]: yeah it might not be quite the right word https://en.wikipedia.org/wiki/Sentinel_value

**Pull request merged**